app/controllers/usages_controller.rb

```diff
@@ -41,6 +41,10 @@ def create
     @usage = @section.usages.build(usage_params)
 
     if @usage.save
+      if @usage.define_device == '1' && @usage.fqdn.present?
+        device_created = Device.create!(name: @usage.fqdn, device_type_id: 1)
+        @usage.update!(device_id: device_created.id)
```

Comment:
When your link is defined between usage ans device you can compact like this:
```suggestion
        @usage.update!(device: Device.create!(name: @usage.fqdn, device_type_id: 1))
```

Reply:
Fixed in next commit